ocis-pkg/registry/registry.go

```diff
@@ -31,11 +31,12 @@ var (
 
 // Config is the config for a registry
 type Config struct {
-	Type         string   `mapstructure:"type"`
-	Addresses    []string `mapstructure:"addresses"`
-	Username     string   `mapstructure:"username"`
-	Password     string   `mapstructure:"password"`
-	DisableCache bool     `mapstructure:"disable_cache"`
+	Type         string        `mapstructure:"type"`
+	Addresses    []string      `mapstructure:"addresses"`
+	Username     string        `mapstructure:"username"`
+	Password     string        `mapstructure:"password"`
+	DisableCache bool          `mapstructure:"disable_cache"`
+	RegisterTTL  time.Duration `mapstructure:"register_ttl"`
```

Comment:
Why not DefaultTTL here? because we are effectively using this as the register TTL when registering a service, not only as the deffault TTL. We ALSO use it as the Default TTL and not only.